<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeSelectWindow</class>
 <widget class="QDialog" name="NodeSelectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(21, 24, 28);
	color: rgb(255,255,255);
}

QFrame{
	border: 0px solid white;
	border-radius: 0px;
	border-color: rgb(49, 68, 101);
}

QScrollBar:vertical{
	width:  12px;
	background-color: rgb(38, 38, 38);
	margin: 0px 0px 0px 0px;
}
QScrollBar:handle{
	min-height: 45px;
	border-radius: 4px;
	background-color: rgb(59, 59, 59);
}
QScrollBar:arrow{
	border-radius: 4px;
	background-color: rgb(69, 69, 69);
}
QScrollBar:handle:hover{
	border-radius: 4px;
	background-color: rgb(79, 79, 79);
}
QScrollBar::sub-page:vertical {
    background-color: rgb(31, 31, 31)
}

QScrollBar::add-page:vertical {
    background-color: rgb(31, 31, 31)
}


QProgressBar{ 
	color: rgba(255,255,255,200);
	border: solid grey;
	border-top-right-radius: 12px;
	border-top-left-radius: 12px;
	border-bottom-right-radius: 12px;
	border-bottom-left-radius: 12px;
	background-color: rgb(15, 25, 45);
}
QProgressBar::chunk {
	background-color:
	qlineargradient(
	spread:pad, x1:0, y1:1, x2:1, y2:0,
	stop:0 rgba(80, 120, 150, 100), 
	stop:1 rgba(80, 120, 150, 225));

	border-top-right-radius: 5px;
	border-top-left-radius: 5px;
	border-bottom-right-radius: 5px;
	border-bottom-left-radius: 5px;
}

QListWidget{
	border: 4px solid rgb(33, 37, 41);
	background-color: rgb(33, 37, 41);
	border-radius: 4px;
	alternate-background-color: rgba(0, 0, 0,25);
}

QListWidget:item:hover{
	alternate-background-color: rgb(255, 255, 127);
	background-color: rgb(43, 46, 94);
}

QListWidget:item:selected{
	color: white;
	alternate-background-color: rgb(255, 255, 127);
	background-color: rgb(43, 46, 94);
}

QListWidget:item:pressed{
	background-color:rgba(155, 155, 155,155);
	alternate-background-color: rgb(155, 155, 155);
}


QMenuBar{
	background-color: rgb(33, 37, 41);
	border: 2px solid rgb(43, 45, 48);
border-left: 0px;
	border-top: 0px;
	border-right: 0px;
}



</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	padding-left: 10px;
	padding-right: 10px;
	color: rgb(205, 205, 205);
	background-color: rgb(82, 82, 82);
	border: 4px solid rgb(82, 82, 82);
	border-radius: 4px;
}

QPushButton:hover{
	border-color: rgb(100, 100, 100);
	background-color: rgb(100, 100, 100);
}
QPushButton:pressed{
	border-color: rgb(100, 100, 100);
	background-color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>2</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>31</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="lineCount">
        <property name="text">
         <string>Items Selected:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QListWidget" name="listWidget">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget::item{
	color: rgba(255, 255, 255, 0);
	background-color: rgba(0,0,0,40);
	border: 3px solid rgb(90, 93, 255);
	border-top: 0px;
	border-bottom: 0px;
	border-right: 0px;
	padding-left: 3px;
}

QListWidget::item:selected{
	color: rgba(255, 255, 255, 0);
	background-color: rgba(90,90,120,130);
	border: 3px solid rgb(90, 93, 255);
	border-top: 0px;
	border-bottom: 0px;
	border-right: 0px;
	padding-left: 3px;
}

QListWidget::item:hover{
	color: rgba(255, 255, 255, 0);
	background-color: rgba(80,80,160,110);
	border: 3px solid rgb(90, 93, 255);
	border-top: 0px;
	border-bottom: 0px;
	border-right: 0px;
	padding-left: 3px;
}</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="flow">
      <enum>QListView::TopToBottom</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>false</bool>
     </property>
     <property name="layoutMode">
      <enum>QListView::SinglePass</enum>
     </property>
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="viewMode">
      <enum>QListView::ListMode</enum>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
     <property name="uniformItemSizes">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="selectionRectVisible">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>New Item</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>New Item</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>New Item</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">
QWidget{
	border: 4px solid rgb(33, 37, 41);
	background-color: rgb(33, 37, 41);
	border-radius: 4px;
	alternate-background-color: rgba(0, 0, 0,25);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="fileLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Source File</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="indent">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="nodeLabel">
              <property name="font">
               <font>
                <pointsize>7</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(158, 162, 242);</string>
              </property>
              <property name="text">
               <string>0 node(s)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="meshLabel">
              <property name="font">
               <font>
                <pointsize>7</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Mesh Count:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="icons.qrc">:/icons/Icons/menu.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NodeSelectWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NodeSelectWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
